PersonDataRequest/PersonDataRequest/Controller/ApiHelper.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ApiHelper.swift
+//  PersonDataRequest
+//
+//  Created by Ivan on 13.02.2024.
+//
+
+import Foundation
+
+struct ApiHelper {
+    
+    func makeRequest<T: Decodable>(urlString: String, t: T.Type, completion: @escaping (T?) -> Void) {
+        guard let url = URL(string: urlString) else {
+            completion(nil)
+            return
+        }
+        
+        let task = URLSession.shared.dataTask(with: url) { (data, response, error) in
+            if let error = error {
+                print("Gender request error: \(error.localizedDescription)")
+                return
```

Comment:
You need to remove this return or call the `completion(nil`) before return

Reply:
I removed it